java-ws/server/Todo/src/main/java/com/couchbase/lite/todo/model/Task.java

```diff
@@ -173,6 +185,8 @@ public static void updateImage(UserContext context,
         MutableDocument mdoc = doc.toMutable();
         mdoc.setValue(KEY_IMAGE, blob);
         db.save(mdoc);
+
+        System.out.println(" Blob Image to JSON (when update image of task): "+ blob.toJSON());
```

Comment:
I don't understand these messages.  Not a blocker, but if you could make them more useful, it would be awesome.